set.mm

```diff
@@ -467069,6 +467273,448 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
+      WSXLXNZTVWPVUOVWQWPVUCVUNRZDUWSUXKUYIVUDXPXQZXRVXQDUYCVUDUXKVUBVVMUYQVXLV
+      UNVWFXSVXMVUNRXTYAVXMVXPNZVVEVUDUYILZUYCVUCVXPVVEVYFMZVXLVUCVXPNZVXSVXTVW
+      PVXPVYGVUCVXSVXPVYATVUCVXTVXPVYBTVWPVYHVWQWPVUCVXPRDUWSUXKUYIVUDYBXQZXRVY
+      EUWSUYCVUDUYIVUBVWGUYQVXLVXPVWOXSVXMVXPRXTYAVXMVUDVVKKVUNVXPYCVXMVUDAVVKV
+      UCVXLRZVUCAVVKMVXLVUCVVKAUWRVWRUXDUXGUYEUXNUYAUYTUYJUYQVWSYEYDZTYFVUDDUWS
+      YGYHYIZVXMAVWHVUDUXEVWJUXGUYEUXNUYAUYTUYJUYQVXLVWLYOVYJYJZVVEUXQUXLUYCVWH
+      EVUDVWKUXLXEZUXQXEZUYCXEZUUAWDVUCUYCAEUXLVVEJUXQVWHBVUDFUXMVXKVUCUWPVWAUW
+      PUWQUWRUXGUYEUXNUYAUYTUYJUYQUUBZVWCWTZUXLUXLMVUCVYNWPVWKVYLVYMIUXMXEZVYOV
+      UCVUTJAVVEPUXMUNVUCJAUYCVUTVUBVVLUYQVWTTZUUEVXAUUOUUCVWPVUCVWQWPZVYKUUFVU
+      CVXGVUHEJUWSVYFVUDUXQOZPZQOZVXFOUXFUYLVXFOZFVUCVXCVUHVXEWUDVXFVUCVXBVUGEQ
+      VUCJDVVFVUFVUOVVEVUEVUDUXQVYDYKYLYMVUCVXDWUCEQVUCJUWSVVFWUBVYHVVEVYFVUDUX
+      QVYIYKYLYMVCVUCUXFVUHUYLWUDVXFVUMVUCUYLUYPWUDVUBUYQRWUCUYOEQJUCUWSWUBUYNV
+      UKVYFUYMVUDUXOUXQVUDUXOUYIVAVULVCVDVEVFVCVUCEUUDKZUXFVWHKWUEFMVUCUWPVWAWU
+      FVYQVWCEUUGUUHVUCUXAVWHUXFUXEUXAVWHUGZUXGUYEUXNUYAUYTUYJUYQUXEVWADVWHUGZW
+      UGVWDUXEDAVWHVVQVWLUUIZDCVWHEVWKHUUJSXJUXIUXFUXAKZUYEUXNUYAUYTUYJUYQUXIUW
+      TUXAUXFUXEUXGRZUUKZVLZYJVWHVXFEUYKUXFFVWKVXHIUYKXEZUULSUUMYNVUCUFUKZUXMUN
+      UOZEJAVUDWUOLZVUDUXQOZPZQOZFMZNZWUOVVBMZYPZVVDVVINZVVJYPUFUYCAUROZVUTWUOV
+      UTMZWVBWVEWVCVVJWVGWUPVVDWVAVVIWUOVUTUXMUNUUNWVGWUTVVHFWVGWUSVVGEQWVGJAWU
+      RVVFWVGWUQVVEVUDUXQVUDWUOVUTUUPYKUUQYMUURUUSWUOVUTVVBUUTUVAUXEWVDUFWVFUVB
+      ZUXGUYEUXNUYAUYTUYJUYQUXEVWAVWJVWBWVHVWDVWLVWEVWAVWJNVWBWVHJVWHUXQUXLUYCF
+      AEUXMUFVWKVYPVYNVYOIVYSUVCWGUVEXJVUCVUTWVFKVVLVYTVUCUYCAVUTWCBUXEUYCWCKUX
+      GUYEUXNUYAUYTUYJUYQVVSXJVXKWFUVDUVFUVGUVHVUCVXSVXTVWPVVAUXKMVYAVYBWUADUWS
+      UXKUYIUVIWDUWRVVCVURMUXDUXGUYEUXNUYAUYTUYJUYQAVUQDUVJYEUVKTUVNVUOVUNVUSUX
+      MMVYCDUXMVUDUXLUMUVOUVLWTYNYKVUOVWAVXOVUPFMVUCVWAVUNVYRTVUODVWHVUDUXEWUHU
+      XGUYEUXNUYAUYTUYJUYQVUNWUIYOVYCYJUXQUXLUXMVWHEVUDFVWKVYNVYOVYSIUVMSYNYLYM
+      VUCEUWEKZDWCKVUJFMVUCVXIWVIVXJEUVPWTVUCUXFCDWUMWMDJEWCFIUVQSUVRYQUXIUYRUD
+      UYSYRZUYEUXNUYAUXIUXEUYLUWTKZWVJUXEUXGUVSZUXIUWTEUVTLKZUXFUWTKWVKUXEVWAUX
+      GUWTEUWFLZKZWVMVWDUXEWVOUXGUXEVWAVWIWVOVWDVWMWVNUWSCVWHEVWKWVNXEZHUWASTWV
+      NUWTEWVPUWBUWCUXIUWTUXAUXFWUKUWDUWTEUYKUXFWUNUWGSUXEWVKWVJUXEUCVWHUXLUXQU
+      YCECUWSUYLUXMUDHVWKVYPVYNVYSVYOVWDVWMYSWGSXSYTYQUXIUXEWUJUYBUBUYDYRZWVLWU
+      LUXEWUJWVQUXEUCVWHUXLUXQUYCECDUXFUXMUBHVWKVYPVYNVYSVYOVWDWUIYSWGSYTUAFUWH
+      UWIUWJUWKUXEFUXBUXEUWTUXAFVWNUXEVWAWUHFUXAKVWDWUIVWHDCEFIVWKHWLSUWLUWMUWN
+      UWO $.
+  $}
+
+  ${
+    $d F b u v w x y $.  $d I b w $.  $d K b u v w x y $.  $d U b u v w x y $.
+    $d V b c k u v w x y $.  $d .0. x y $.
+    dimkerim.0 $e |- .0. = ( 0g ` U ) $.
+    dimkerim.k $e |- K = ( V |`s ( `' F " { .0. } ) ) $.
+    dimkerim.i $e |- I = ( U |`s ran F ) $.
+    $( Given a homomorphism of vector spaces ` F ` over ` V ` , the dimension
```

Comment:
Maybe it is better to reformulate the first sentence as
> Given a linear map ` F ` between vector spaces ` V ` and ` U `, the dimension ...

I think it's more common to call field-preserving homomorphisms between vector spaces "linear maps".